templates/form/input--radio.html.twig

```diff
@@ -10,5 +10,10 @@
  * @see template_preprocess_input()
  */
 #}
-
-<input{{ attributes.addClass('ecl-radio__input').addClass('ecl-u-sr-only') }} />{{ children }}
+{% include '@ecl/ecl-forms-radios' with {
+    'id': attributes.id,
+    'name': attributes.name,
+    'value': attributes.value,
+    'checked': attributes.checked,
+    'label': title
+} %}
```

Comment:
According to https://ec-europa.github.io/europa-component-library/components/detail/ecl-forms-radios, some parameters are missing:
    - "is_disabled" (boolean): is disabled (optional, default: false)
    - "has_error" (boolean): has error (optional, default: false)
    - "extra_class" (string): extra CSS classes (optional, default: '')